<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mainICo.png</normaloff>mainICo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    color: #eaeaea;
    background-color: #2b2b2b;
    selection-background-color: #3daee9;
    selection-color: #eaeaea;
}

QHeaderView::section {
    background-color: #3b3b3b;
    color: #eaeaea;
    padding: 5px;
}

QTabWidget::pane {
    border: 1px solid #222222;
    background-color: #2b2b2b;
}

QTabBar::tab {
    background-color: #2b2b2b;
    color: #eaeaea;
    padding: 8px;
    margin-right: 1px;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background-color: #3daee9;
    color: #eaeaea;
}

QLineEdit, QTextEdit, QPlainTextEdit, QListView, QTableView, QTreeView {
    background-color: #333333;
    color: #eaeaea;
    selection-background-color: #3daee9;
    selection-color: #eaeaea;
    border: 1px solid #555555;
    border-radius: 2px;
    padding: 2px;
}

QProgressBar {
    border: 1px solid #222222;
    border-radius: 5px;
    background-color: #555555;
    color: #eaeaea;
}

QProgressBar::chunk {
    background-color: #3daee9;
    width: 5px;
    margin: 0.5px;
}

QPushButton {
    color: #eaeaea;
    background-color: #333333;
    border: 1px solid #555555;
    padding: 5px;
    border-radius: 2px;
}

QPushButton:hover {
    background-color: #3daee9;
    color: #eaeaea;
}

QToolButton {
    color: #eaeaea;
    background-color: #333333;
    border: 1px solid #555555;
    padding: 5px;
    border-radius: 2px;
}

QToolButton:hover, QToolButton:checked {
    background-color: #3daee9;
    color: #eaeaea;
}

QComboBox {
    color: #eaeaea;
    background-color: #333333;
    border: 1px solid #555555;
    padding: 2px;
    border-radius: 2px;
    min-width: 6em;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #555555;
    border-left-style: solid;
}

QComboBox::down-arrow {
    image: url(arrow-down.png);
    width: 10px;
    height: 10px;
}

QComboBox::down-arrow:hover {
    image: url(arrow-down-hover.png);
}

QScrollBar {
    border: 1px solid #222222;
    background-color: #2b2b2b;
}

QScrollBar:horizontal {
    height: 15px;
    margin: 3px 15px 3px 15px;
}

QScrollBar:vertical {
width: 15px;
margin: 15px 3px 15px 3px;
}

QScrollBar::handle {
background-color: #555555;
border-radius: 5px;
}

QScrollBar::handle:hover {
background-color: #3daee9;
}

QScrollBar::add-page, QScrollBar::sub-page {
background-color: #2b2b2b;
}

QScrollBar::add-line, QScrollBar::sub-line {
border: 1px solid #222222;
background-color: #555555;
}

QCheckBox, QRadioButton {
color: #eaeaea;
}

QCheckBox::indicator, QRadioButton::indicator {
width: 15px;
height: 15px;
border: 1px solid #555555;
border-radius: 2px;
background-color: #333333;
}

QCheckBox::indicator:disabled, QRadioButton::indicator:disabled {
border-color: #777777;
background-color: #444444;
}

QCheckBox::indicator:checked, QRadioButton::indicator:checked {
background-color: #3daee9;
border-color: #3daee9;
}

QMenuBar {
background-color: #2b2b2b;
color: #eaeaea;
}

QMenuBar::item {
padding: 2px 10px 2px 10px;
background-color: transparent;
}

QMenuBar::item:selected {
background-color: #3daee9;
color: #eaeaea;
}

QMenu {
background-color: #2b2b2b;
border: 1px solid #222222;
}

QMenu::item {
background-color: transparent;
padding: 2px 20px 2px 20px;
}

QMenu::item:selected {
background-color: #3daee9;
color: #eaeaea;
}

QTreeView::item:selected {
background-color: #3daee9;
color: #eaeaea;
}

QTreeView::item:hover {
background-color: #3daee9;
color: #eaeaea;
}

QHeaderView::section:pressed {
background-color: #3daee9;
}

QTableView::item:selected, QListView::item:selected, QTreeView::item:selected {
background-color: #3daee9;
color: #eaeaea;
}

QTableView::item:hover, QListView::item:hover, QTreeView::item:hover {
background-color: #3daee9;
color: #eaeaea;
}</string>
  </property>
  <widget class="QLineEdit" name="userName">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userPassword">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>150</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Text</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#245cc1;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; color:#7ba1e8;&quot;&gt;ritical &lt;/span&gt;&lt;span style=&quot; color:#245cc1;&quot;&gt;A&lt;/span&gt;&lt;span style=&quot; color:#7ba1e8;&quot;&gt;lert &lt;/span&gt;&lt;span style=&quot; color:#245cc1;&quot;&gt;M&lt;/span&gt;&lt;span style=&quot; color:#7ba1e8;&quot;&gt;anagement &lt;/span&gt;&lt;span style=&quot; color:#245cc1;&quot;&gt;L&lt;/span&gt;&lt;span style=&quot; color:#7ba1e8;&quot;&gt;ogin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
